docs/source/execution_device_interoperability.ipynb

```diff
@@ -0,0 +1,202 @@
+{
```

Comment:
<p>Nit: ... can also be set at the global level with the ...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cuml/pull/5130/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>